crates/wasm-mutate/src/mutators/mod.rs

```diff
@@ -31,3 +31,32 @@ pub mod peephole;
 pub mod remove_export;
 pub mod rename_export;
 pub mod snip_function;
+
+// macro for mutation assesment
+#[cfg(test)]
+#[macro_export]
+macro_rules! match_mutation {
+    (
+        $original: tt, $mutator: expr, $expected: tt
+    ) => {{
```

Comment:
I think this can be a function, and doesn't need to be a macro.

```rust
fn assert_mutation(mutator: impl Mutate, original: &str, expected: &str) { ... }

Reply:
Gotcha